Simulation/Pathfinding/HighLevelCBS.cpp

```diff
@@ -16,29 +21,37 @@ Solution HighLevelCBS::findSolution(std::shared_ptr<Graph> graph, std::vector<st
     /**
      * While OPEN not empty do
      */
+    int iterations = 0;
     while (open.size() > 0) {
```

Comment:
! open.empty() is probably faster

Reply:
turns out empty is only faster if implementation is changed to lists, where its up to the implementation which is faster